update/files/multiFileReader.go

```diff
@@ -120,6 +111,25 @@ func (m *multiFileReader) ReadNextItem(fileName string) (string, []byte, error)
 	return string(key), value, nil
 }
 
+func (m *multiFileReader) getDataReader(fileName string) (update.DataReader, error) {
+	scanner, ok := m.dataReader[fileName]
+	if !ok {
+		return nil, update.ErrNilDataReader
+	}
+
+	canRead := scanner.Scan()
+	if canRead {
+		return scanner, nil
+	}
+
+	err := scanner.Err()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
maybe add a comment that scanner returns nil is it is endOfFile.

Reply:
done.